tests/floris_interface_test.py

```diff
@@ -23,15 +23,90 @@ def test_calculate_wake():
     that the yaw angles are correctly set in subsequent calls to calculate_wake.
     """
     fi = FlorisInterface(configuration=YAML_INPUT)
-    yaw_angles = 20 * np.ones((fi.floris.flow_field.n_wind_directions, fi.floris.flow_field.n_wind_speeds, fi.floris.farm.n_turbines))
+    yaw_angles = 20 * np.ones(
+        (
+            fi.floris.flow_field.n_wind_directions,
+            fi.floris.flow_field.n_wind_speeds,
+            fi.floris.farm.n_turbines
+        )
+    )
     fi.calculate_wake(yaw_angles=yaw_angles)
     assert fi.floris.farm.yaw_angles == yaw_angles
 
-    yaw_angles = np.zeros((fi.floris.flow_field.n_wind_directions, fi.floris.flow_field.n_wind_speeds, fi.floris.farm.n_turbines))
+    yaw_angles = np.zeros(
+        (
+            fi.floris.flow_field.n_wind_directions,
+            fi.floris.flow_field.n_wind_speeds,
+            fi.floris.farm.n_turbines
+        )
+    )
     fi.calculate_wake(yaw_angles=yaw_angles)
     assert fi.floris.farm.yaw_angles == yaw_angles
 
 
+def test_calculate_no_wake():
+    """
+    In FLORIS v3.2, running calculate_no_wake twice incorrectly set the yaw angles when the first
```

Comment:
Heads up to update this docstring

Reply:
Updated, but still left a good portion in, as `calculate_no_wake` suffered from the same issue.